src/System.Console/tests/Color.cs

```diff
@@ -92,6 +92,68 @@ public static void NonRedirectedOutputDoesUseAnsiSequences()
         Assert.Equal(3, Encoding.UTF8.GetString(data.ToArray()).ToCharArray().Count(c => c == Esc));
     }
 
+    [Fact]
+    [PlatformSpecific(PlatformID.AnyUnix)]
+    public void VerifyInstalledTermInfosParse()
+    {
+        foreach (string location in ConsolePal.TermInfo.Database._terminfoLocations)
+        {
+            // Skip directories that dont exist
+            if (Directory.Exists(location) == false)
+                continue;
+
+            foreach (string term in Directory.EnumerateFiles(location, "*", SearchOption.AllDirectories))
+            {
+                if (term.ToUpper().Contains("README")) continue;
+                ConsolePal.TermInfo.Database db = ConsolePal.TermInfo.Database.ReadDatabase(Path.GetFileName(term));
+                Assert.True(db != null, string.Format("Could not parse terminfo file {0}", term)); // Don't use Assert.NotNull so we can print out the term info file on failure
```

Comment:
As an alternative, you could consider exposing the results of enumerating the files as a MemberData function, e.g. (pseudo-code)
```C#
public static IEnumerable<object[]> Terms()
{
    var results = new List<object[]>();
    foreach (string location in ConsolePal.TermInfo.Database._termInfoLocations)
    foreach (string termPath in Directory.EnumerateFiles(location, "*", SearchOptions.AllDirectories))
    {
        string term = Path.GetFileName(termPath);
        string dirName = Path.GetFileName(Path.GetDirectoryName(termPath));
        if (ter[0] == dirName[0]) // terminfo files are stored in a directory beginning with the same letter
            results.Add(new object[] { term });
    }
    Assert.True(results.Count > 0);
    return results;
}
```
which you can then consume in a theory:
```C#
[Theory]
[PlatformSpecific(PlatformID.AnyUnix)]
public void VerifyInstalledTermInfosParse(string term)
{
    ConsolePal.TermInfo.Database db = ConsolePal.TermInfo.Database.ReadDatabase(term);
    Assert.NotNull(db);
    ... // additional verification on db here (see later comment)
}
```